scripts/sentry-notifications.coffee

```diff
@@ -0,0 +1,83 @@
+# Description:
+#   Posts Hangouts Notifications from AgileVentures to Slack and Gitter.
+#
+# Dependencies:
+#   "requestify": "*"
+#
+# Configuration:
+#
+# Commands:
+#   post hangout title, link and type to /hubot/hangouts-notify
+#   post hangout title, video_lin and type to /hubot/hangouts-video-notify
+#
+# Author:
+#   sampritipanda
+
+
+[CHANNELS, GITTER_ROOMS] = require('./../config/' + process.env.LIVE_ENV + '.coffee')
+
+request = require('request')
+rollbar = require('rollbar')
+
+rollbar.init(process.env.ROLLBAR_ACCESS_TOKEN, {enabled: process.env.ENABLE_ROLLBAR != 'false'})
+
+module.exports = (robot) ->
+  send_slack_message = (channel, message, user) ->
+    request.post 'https://slack.com/api/chat.postMessage', form:
+      channel: channel
+      text: message
+      username: user.name
+      icon_url: user.avatar
+      parse: 'full'
+      token: process.env.SLACK_API_TOKEN
+    , (error, response, body) ->
+      try payload = JSON.parse body
+      catch e then payload = {}
+      unless payload['ok']
+        rollbar.reportMessageWithPayloadData payload['error'],
+          origin: 'send_slack_message'
+          level: 'error'
+          custom:
+            error: error
+            response: response
+            body: body
+
+  robot.router.post "/hubot/sentry-notify/:room", (req, res) ->
+    # {
```

Comment:
ah, you're saying it's an example of what we receive?  ah good to have then - yeah, add a note to that effect i guess